client/state/ui/reducer.js

```diff
@@ -31,6 +30,8 @@ import preview from './preview/reducer';
 import reader from './reader/reducer';
 import route from './route/reducer';
 import themeSetup from './theme-setup/reducers';
+import masterbarVisibility from './masterbar-visibility/reducer';
+import section from './section/reducer';
```

Comment:
Another opportunity for zero-risk separate PR. Makes the main diff much smaller.

Reply:
I'll pull this out too.

Reply:
Done! See #35092.